docs/install/install-eventing-with-yaml.md

```diff
@@ -0,0 +1,225 @@
+   {{% tab name="Apache Kafka Broker" %}}
+
+The following commands install the Apache Kafka broker, and run event routing in a system namespace,
+`knative-eventing`, by default.
+
+1. Install the Kafka controller by entering the following command:
+
+    ```bash
+    kubectl apply -f {{< artifact org="knative-sandbox" repo="eventing-kafka-broker" file="eventing-kafka-controller.yaml" >}}
+    ```
+
+1. Install the Kafka Broker data plane by entering the following command:
+
+    ```bash
+    kubectl apply -f {{< artifact org="knative-sandbox" repo="eventing-kafka-broker" file="eventing-kafka-broker.yaml" >}}
+    ```
+
+For more information, see the [Kafka Broker](./../eventing/broker/kafka-broker.md) documentation.
+{{< /tab >}}
+
+   {{% tab name="MT-Channel-based" %}}
+
+The following command installs an implementation of Broker that utilizes
+Channels and runs event routing components in a System Namespace, providing a
+smaller and simpler installation.
+
+```bash
+kubectl apply -f {{< artifact repo="eventing" file="mt-channel-broker.yaml" >}}
+```
+
+To customize which broker channel implementation is used, update the following
+ConfigMap to specify which configurations are used for which namespaces:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-br-defaults
+  namespace: knative-eventing
+data:
+  default-br-config: |
+    # This is the cluster-wide default broker channel.
+    clusterDefault:
+      brokerClass: MTChannelBasedBroker
+      apiVersion: v1
+      kind: ConfigMap
+      name: imc-channel
+      namespace: knative-eventing
+    # This allows you to specify different defaults per-namespace,
+    # in this case the "some-namespace" namespace will use the Kafka
+    # channel ConfigMap by default (only for example, you will need
+    # to install kafka also to make use of this).
+    namespaceDefaults:
+      some-namespace:
+        brokerClass: MTChannelBasedBroker
+        apiVersion: v1
+        kind: ConfigMap
+        name: kafka-channel
+        namespace: knative-eventing
+```
+
+The referenced `imc-channel` and `kafka-channel` example ConfigMaps would look
+like:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: imc-channel
+  namespace: knative-eventing
+data:
+  channelTemplateSpec: |
+    apiVersion: messaging.knative.dev/v1
+    kind: InMemoryChannel
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kafka-channel
+  namespace: knative-eventing
+data:
+  channelTemplateSpec: |
+    apiVersion: messaging.knative.dev/v1alpha1
+    kind: KafkaChannel
+    spec:
+      numPartitions: 3
+      replicationFactor: 1
+```
+
+**NOTE:** In order to use the KafkaChannel make sure it is installed on the cluster as discussed above.
+
+{{< /tab >}}
+
+{{< /tabs >}}
+
+
+## Next steps
+
+After installing Knative Eventing:
+
+- If you want to add extra features to your installation, see [Installing optional extensions](./install-extensions.md).
```

Comment:
I'd make all the channels and broker stuff that aren't part of core eventing optional steps too instead of including them above

Reply:
As in putting them on the optional topic? The way I imagined it was a serving topic and eventing topic and and and optional extensions topic. Are the broker and channels stuff optional extensions? Or do you think I should change the topic name from "Installing optional extensions" to something else?

Reply:
If you are recommending splitting the optional topic into "optional for serving" and "optional for eventing", I am concerned about creating clutter in the subnav. Splitting further will mean that we will end up with four topics (five including the Prerequisites) where there was only one before. 

If we agree that splitting the optional topic is a good idea, I think we should reorganise the subnav.